rpc/rpc-interfaces.go

```diff
@@ -12,5 +12,5 @@ type Client interface {
 
 // Server describes the functions an RPC Server performs
 type Server interface {
-	Handle(string, func([]byte) ([]byte, error))
```

Comment:
That's correct, and currently nothing does. If we decide we need things outside the package to implement it we can change it back. Or I can leave the signature as it currently is in master. This seemed a little clearer to me, but I don't feel strongly about it.